<launch>

<node pkg="rviz" type="rviz" name="rviz_nav" output="screen" args="-d $(find mrpt_rbpf_slam)/rviz/rviz_conf.rviz">
    </node>

       <param name="rawlog_play_delay" value="10"/> 

  <param name="ini_filename" value="$(find mrpt_rbpf_slam)/tutorial/grid_slam_demo.ini"/>
  <!--<param name="rawlog_filename" value="$(find mrpt_rbpf_slam)/tutorial/grid_slam_demo_2.rawlog"/>-->


  <param name="odom_frame_id" value="/odom"/>
  <param name="global_frame_id" value="/map"/>
  <param name="base_frame_id" value="np"/>
  <param name="sensor_source" value="/scan_front"/><!-- for multiple sensors, place comma eg "r1/front_laser/scan,/beacon" -->


  <node pkg="mrpt_rbpf_slam" type="mrpt_rbpf_slam" name="mrpt_rbpf_slam" output="screen">
  </node>
 
  <!--  <include file="$(find mrpt_rbpf_slam)/launch/demo_play.launch" />-->
</launch>

